GoogleService-Info.plist

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
As a company handling financial data, you have to make sure not to expose valuable data such as this or make it easily viewable -- using this someone could in theory write to the db, try to access the client and have all the necessary components for it. 

I suggest you look into environment variables, which are used to hide keys likes the ones below